assets/js/src/form_editor/store/map_form_data_after_loading.jsx

```diff
@@ -8,18 +8,22 @@ export default function mapFormDataAfterLoading(data) {
       ...data.settings,
       placeFormBellowAllPages: data.settings.place_form_bellow_all_pages === '1',
       placeFormBellowAllPosts: data.settings.place_form_bellow_all_posts === '1',
+      placementBellowAllPostsEnabled: data.settings.form_placement_bellow_posts_enabled === '1',
```

Comment:
It would be nice to add these new props mappings into a test so that someone doesn't remove them in some refactoring. The test should also test that in case these are not set they are mapped as false.